<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceRunDistancePage</class>
 <widget class="QWidget" name="MaintainceRunDistancePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
	color:rgb(23,176,176);
	border:2px solid white;
font: 28px &quot;微软雅黑&quot;
}</string>
  </property>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>530</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
		border-radius:8px;
background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>520</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="StartTImelbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RundistanceADaylbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>138</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RuntimeADaylbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>196</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RundistanceAcclbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>254</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>191</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:22px;
</string>
   </property>
   <property name="text">
    <string>起始时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>138</y>
     <width>191</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:22px

</string>
   </property>
   <property name="text">
    <string>日行驶距离
（千米/天）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>196</y>
     <width>191</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:22px

</string>
   </property>
   <property name="text">
    <string>日行驶时间
（分/天）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>254</y>
     <width>191</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:22px

</string>
   </property>
   <property name="text">
    <string>合计行驶距离
（千米）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>returnBtn</zorder>
  <zorder>StartTImelbl</zorder>
  <zorder>RundistanceADaylbl</zorder>
  <zorder>RuntimeADaylbl</zorder>
  <zorder>RundistanceAcclbl</zorder>
  <zorder>lbl</zorder>
  <zorder>lbl2</zorder>
  <zorder>lbl2_2</zorder>
  <zorder>lbl2_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
